crypto/rand/rand_vms.c

```diff
@@ -32,10 +32,21 @@
 #  pragma message disable DOLLARID
 # endif
 
+# include <dlfcn.h>              /* SYS$GET_ENTROPY presence */
+
 # ifndef OPENSSL_RAND_SEED_OS
 #  error "Unsupported seeding method configured; must be os"
 # endif
 
+/*
+ * DATA COLLECTION METHOD
+ * ======================
+ *
+ * This is a very low entropy method to get *some* entropy.
```

Comment:
This is a method to get *some* very low quality entropy ??

Reply:
Much better phrasing, thank you.  Applied